schema/geometry_schema.json

```diff
@@ -1,5 +1,6 @@
 {
 	"$schema": "http://json-schema.org/draft-04/schema#",
+	"version": "0.3.0",
```

Comment:
Here is a pull request that includes a version property https://github.com/NREL/openstudio-geometry-editor/pull/175